pages/k8s/release-notes.md

```diff
@@ -45,16 +45,63 @@ Existing deployments that are upgraded to **CDK 1.14** will
 continue to use **KubeDNS** until the operator chooses to upgrade to
 **CoreDNS**. See the [upgrade notes][upgrade-notes] for details.
 
-- Notable feature xxxxx xx
-
-xxxx xxx xxxx xxx xxxx xx xxx xxxx xxx xx xxx xxxx xxxx xxxx xxx xx xxxx xxxx xxxxxx xx
-xxxxxx xxx xxx xx xxxx xxxx  [documentation][doc-link].
+- one-liner xxxxxx xxxxx xxx
+- one-liner xxx x xx x xx xxx
 
 ## Fixes
 
-- Fixed xxxx xxx xx xx xxx xxxxxxx xxxx ([Issue](https://bugs.launchpad.net/charmed-kubernetes/))
-- Fixed xxxx xxx xxxxx xx xx xxxxx xxxx ([Issue](https://bugs.launchpad.net/charmed-kubernetes/))
-- Fixed xx x xxx x xx xxx xxx xxxx xx xx ([Issue](https://bugs.launchpad.net/charmed-kubernetes/))
+ - Added an action to upgrade Docker ([Issue](https://github.com/juju-solutions/layer-docker/pull/135))
+ - Added better multi-client support to EasyRSA ([Issue](https://github.com/juju-solutions/layer-easyrsa/pull/15))
+ - Added block storage config options ([Issue](https://github.com/juju-solutions/charm-openstack-integrator/pull/15))
+ - Added block storage options for OpenStack ([Issue](https://github.com/juju-solutions/kubernetes/pull/218))
+ - Added dashboard-auth config option to master ([Issue](https://github.com/juju-solutions/kubernetes/pull/222))
+ - Added dashboard login using kubeconfig ([Issue](https://github.com/juju-solutions/cdk-addons/pull/72))
+ - Added docker registry handling to master ([Issue](https://github.com/juju-solutions/kubernetes/pull/210))
+ - Added more TLS options to Docker registry ([Issue](https://github.com/CanonicalLtd/docker-registry-charm/pull/20))
+ - Added new folder/respool_path config for vSphere ([Issue](https://github.com/juju-solutions/charm-vsphere-integrator/pull/2))
+ - Added proxy support to Docker registry ([Issue](https://github.com/CanonicalLtd/docker-registry-charm/pull/21))
+ - Added read-only mode for Docker registry ([Issue](https://github.com/CanonicalLtd/docker-registry-charm/pull/22))
+ - Added token auth option for dashboard ([Issue](https://github.com/juju-solutions/cdk-addons/pull/77))
```

Comment:
^ ditto

Reply:
This one I think we should leave out of "Features". It's too minor in my opinion.